--- libxslt-1.0.7/configure.in~	Wed Nov 14 19:54:56 2001
+++ libxslt-1.0.7/configure.in	Wed Nov 14 19:54:56 2001
@@ -106,49 +106,6 @@
 AM_CONDITIONAL(WITH_PERL, test "$PERL" != "false")
 
 dnl
-dnl Debug for DV (-Wunreachable-code)
-dnl
-if test "${LOGNAME}" = "veillard" -a "`pwd`" = "/u/veillard/XSLT" ; then
-    if test "${with_mem_debug}" = "" ; then
-	with_mem_debug="yes"
-    fi
-    CFLAGS="-Wall -g -O -ansi -pedantic -W -Wunused -Wimplicit -Wreturn-type -Wswitch -Wcomment -Wtrigraphs -Wformat -Wchar-subscripts -Wuninitialized -Wparentheses -Wshadow -Wpointer-arith -Wcast-align -Wwrite-strings -Waggregate-return -Wstrict-prototypes -Wmissing-prototypes -Wnested-externs -Winline "
-fi
-
-AC_ARG_WITH(debug, [  --with-debug            Add the debugging code (on)])
-if test "$with_debug" = "no" ; then
-    echo Disabling debug support
-    WITH_XSLT_DEBUG=0
-else    
-    WITH_XSLT_DEBUG=1
-fi
-AC_SUBST(WITH_XSLT_DEBUG)
-
-AC_ARG_WITH(mem_debug, [  --with-mem-debug        Add the memory debugging module (off)])
-if test "$with_mem_debug" = "yes" ; then
-    echo Enabling memory debug support
-    WITH_MEM_DEBUG=1
-else    
-    WITH_MEM_DEBUG=0
-fi
-AC_SUBST(WITH_MEM_DEBUG)
-
-dnl 
-dnl Is debugger support requested
-dnl
-AC_ARG_WITH(with_debugger, [  --with-debugger        Add the debugging module (off)])
-if test "$with_debugger" = "no" ; then
-    echo Disabling debugger
-    WITH_DEBUGGER=0
-else
-    echo Enabling debugger
-    WITH_DEBUGGER=1
-    AC_DEFINE(WITH_DEBUGGER)
-fi
-AM_CONDITIONAL(WITH_DEBUGGER, test "${WITH_DEBUGGER}" = "1")
-AC_SUBST(WITH_DEBUGGER)
-
-dnl
 dnl The following new parameters were added to offer
 dnl the ability to specify the location of the libxml
 dnl library during linking and compilation.
--- libxslt-1.0.7/xsltproc/Makefile.am~	Wed Nov 14 19:05:41 2001
+++ libxslt-1.0.7/xsltproc/Makefile.am	Wed Nov 14 19:05:41 2001
@@ -11,22 +11,6 @@
 xsltproc_DEPENDENCIES = $(DEPS)
 xsltproc_LDADD = $(LDADDS)
 
-if WITH_DEBUGGER
-DEPS = $(top_builddir)/libxslt/libxslt.la \
-	$(top_builddir)/breakpoint/libxsltbreakpoint.la \
-	$(top_builddir)/libexslt/libexslt.la 
-else
-DEPS = $(top_builddir)/libxslt/libxslt.la \
-	$(top_builddir)/libexslt/libexslt.la 
-endif
-
-if WITH_DEBUGGER
-LDADDS = $(top_builddir)/libxslt/libxslt.la \
-         $(top_builddir)/libexslt/libexslt.la \
-	$(top_builddir)/breakpoint/libxsltbreakpoint.la \
-	@LIBXML_LIBS@ $(EXTRA_LIBS)
-else
 LDADDS = $(top_builddir)/libxslt/libxslt.la \
          $(top_builddir)/libexslt/libexslt.la \
 	@LIBXML_LIBS@ $(EXTRA_LIBS)
-endif
--- libxslt-1.0.7/libxslt/xsltconfig.h.in~	Wed Nov 14 21:34:28 2001
+++ libxslt-1.0.7/libxslt/xsltconfig.h.in	Wed Nov 14 21:34:28 2001
@@ -35,48 +35,6 @@
 #define LIBXSLT_VERSION_STRING "@LIBXSLT_VERSION_NUMBER@"
 
 /**
- * WITH_XSLT_DEBUG:
- *
- * Activate the compilation of the debug reporting. Speed penalty
- * is insignifiant and being able to run xsltpoc -v is useful. On
- * by default unless --without-debug is passed to configure
- */
-#if @WITH_XSLT_DEBUG@
-#define WITH_XSLT_DEBUG
-#endif
-
-#if @WITH_MEM_DEBUG@
-/**
- * DEBUG_MEMORY:
- *
- * should be activated only when debugging libxslt. It replaces the
- * allocator with a collect and debug shell to the libc allocator.
- * Use configure --with-mem-debug to activate it on both library
- */
-#define DEBUG_MEMORY
-
-/**
- * DEBUG_MEMORY_LOCATION:
- *
- * should be activated only when debugging libxslt.
- * DEBUG_MEMORY_LOCATION should be activated only when libxml has
- * been configured with --with-debug-mem too
- */
-#define DEBUG_MEMORY_LOCATION
-#endif
-
-/**
- * WITH_XSLT_DEBUGGER:
- *
- * Activate the compilation of the debugger support. Speed penalty
- * is insignifiant.
- * On by default unless --without-debugger is passed to configure
- */
-#if @WITH_DEBUGGER@
-#define WITH_XSLT_DEBUGGER
-#endif
-
-/**
  * ATTRIBUTE_UNUSED:
  *
  * This macro is used to flag unused function parameters to GCC
