--- libxslt-1.1.34/xslt-config.in.orig	2019-10-23 19:36:39.000000000 +0200
+++ libxslt-1.1.34/xslt-config.in	2020-01-19 17:55:21.321737264 +0100
@@ -88,7 +88,7 @@
             libs="@XSLT_LIBS@ @XSLT_PRIVATE_LIBS@"
         fi
 
-        if [ "@XSLT_LIBDIR@" != "-L/usr/lib" -a "@XSLT_LIBDIR@" != "-L/usr/lib64" ]; then
+        if [ "@XSLT_LIBDIR@" != "-L/usr/lib" -a "@XSLT_LIBDIR@" != "-L/usr/lib64" -a "@XSLT_LIBDIR@" != "-L/usr/libx32" ]; then
             libs="@XSLT_LIBDIR@ $libs"
         fi
 
